backend/monitoring/dashboard.py

```
@@ -697,15 +738,420 @@ async def get_integration_health(integration_point: str, admin=Depends(get_curre
+                )
+        finally:
+            await db_manager.release_connection(conn)
+    except Exception as e:
+        return StandardResponse(
+            status="error",
+            message=f"Failed to get test status: {str(e)}",
+            data={}
+        )
+
+@router.get("/test-sessions")
+async def get_test_sessions(admin: dict = Depends(get_current_admin_dependency)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)